CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 * None.
 
 ### Enhancements
-* None.
+* Reducing the binary size for Android dependency . (Issue [#616](https://github.com/realm/realm-kotlin/issues/216)).
```

Comment:
```suggestion
* Reducing the binary size for Android dependency . (Issue [#216](https://github.com/realm/realm-kotlin/issues/216)).
```

Reply:
Would be nice to see how the Kotlin `.so` compare to the one we have in Android. That might show if we are forgetting something?

Reply:
let's find out when we link against [STL](https://github.com/realm/realm-kotlin/issues/710)